### PR TITLE
Fix running from PATH and add better usage/manpage

### PR DESCRIPTION
**Pull request by Diatrus**

### CODE REVIEW

File: snapUtil.c [hunk 1]
```diff
@@ -20,13 +20,12 @@ void
...
-	(void) fprintf(stderr, "\t%s -d <snap> <vol>\n", g_pname);
-	(void) fprintf(stderr, "\t%s -r <snap> <vol>\n", g_pname);
-	(void) fprintf(stderr, "\t%s -s <snap> <vol> <mntpnt>\n", g_pname);
-	(void) fprintf(stderr, "\t%s -u <snap> <vol>\n", g_pname);
+	(void) fprintf(stderr, "\t%s -l <vol>\t\t\t(List all snapshots)\n", g_pname);
+	(void) fprintf(stderr, "\t%s -c <snap> <vol>\t\t(Create snapshot)\n", g_pname);
+	(void) fprintf(stderr, "\t%s -n <snap> <newname> <vol>\t(Rename snapshot)\n", g_pname);
+	(void) fprintf(stderr, "\t%s -d <snap> <vol>\t\t(Delete snapshot)\n", g_pname);
+	(void) fprintf(stderr, "\t%s -r <snap> <vol>\t\t(Revert to snapshot)\n", g_pname);
+	(void) fprintf(stderr, "\t%s -s <snap> <vol> <mntpnt>\t(Mount snapshot)\n", g_pname);
```
Comment (mroi) [inline snapUtil.c]: I love the added documentation, but I think this code assumes a certain width for tabs. The alignment could look weird on systems with a different tab width.

Comment (Diatrus) [reply snapUtil.c]: Fixed in 944a3665e95a491380138b0fb77177f804a3a2f1

**Comment by ahl** [comment]
thanks and sorry for the long delay

**Comment by CRKatri** [comment]
> thanks and sorry for the long delay

Thanks for reviewing.
Do you think we could get a new release with all these changes? Unless you had other things you wanted to fix/change but to be honesty this tool seems kinda complete already.

**Comment by ahl** [comment]
I forgot that I had even tagged releases. Suggestions on a version number? It looks like I was using the contemporary macOS version.. maybe 12.1?

**Comment by CRKatri** [comment]
That would work. Thanks

**Comment by CRKatri** [comment]
Just wondering if you could tag it soon.

**Comment by ahl** [comment]
Done

**Comment by CRKatri** [comment]
Thank you.

**Pull request merged**